src/pages/support-your-team.tsx

```diff
@@ -66,7 +66,7 @@ const Support: NextPage<SupportPageProps> = ({ pageData }) => {
           <ButtonAsLink
             $mt={32}
             $mb={92}
-            page={"teacher-hub"}
```

Comment:
It would appear that these link destinations are still hardcoded, which is not what the ticket asks for - can you clarify? Also there is a 'Get started' CTA which points to `https://thenational.academy/develop-your-curriculum` but this landing page is being retired and redirected to `/teachers/curriculum` so it would be better if (assuming we don't make these button CMSable) they pointed to the new curriculum landing page.

Reply:
Thanks James, sorry should have updated the ticket but we discussed in stand up this morning and as we are not currently set up in sanity CMS to provide a customisable link for this field we decided we would hard code it to point to the correct place for now and add a new ticket to update the sanity schema and pull it through.

I am a bit confused about the 'Get started' curriculum link though, it is correct in Sanity and should already be pulling through as `teachers/curriculum`, so I will look into why this isn't working properly, thanks for raising!

Reply:
Ok so I think that the 'Get started' link has not been republished in Sanity so it's still pointing at `/develop-your-curriculum` just waiting for confirmation on that to see if it comes through correctly after publishing

Reply:
@njblore Did you get any further on this? It's still linking to the original landing page

Reply:
The updated URL is now pulling through, as expected.